src/vs/base/browser/ui/splitview/splitview.ts

```diff
@@ -675,6 +675,7 @@ export class SplitView<TLayoutContext = undefined> extends Disposable {
 
 		// Add view
 		const container = $('.split-view-view');
+		container.style.overflow = 'hidden';
```

Comment:
Let's stick it within the terminal component instead of the general split view widget as I'm worried about regression other parts of the workbench that use it. terminal.css seems like an ideal place for this

Reply:
Thank you, I fixed. https://github.com/microsoft/vscode/pull/115345/commits/a916ce9076cb1f84e4f35728a7e4ed7be388adce


https://user-images.githubusercontent.com/17052177/106344480-69fea780-62ed-11eb-8d50-88e990dab837.mp4